packages/contentstack-import/src/import/modules/content-types.ts

```diff
@@ -250,4 +255,44 @@ export default class ContentTypesImport extends BaseClass {
     apiOptions.apiData = globalFieldPayload;
     return apiOptions;
   }
+
+  async updatePendingExtensions(): Promise<any> {
+    let apiContent = fsUtil.readFile(this.extPendingPath) as Record<string, any>[];
+    if (apiContent.length === 0) {
+      log(this.importConfig, `No Extension are present to be updated`, 'success');
```

Comment:
@cs-raj , Please check text added or modified with the Doc team.

Reply:
Yes